vaadin-dialog-flow-demo/src/main/java/com/vaadin/flow/component/dialog/demo/DialogView.java

```diff
@@ -67,7 +68,7 @@ private void addBasicDialog() {
     }
 
     private void addConfirmationDialog() {
-        NativeButton button = new NativeButton(BUTTON_CAPTION);
+        Button button = new Button(BUTTON_CAPTION);
```

Comment:
You can keep this button unchanged. Only the buttons in the dialog need to be updated.

Reply:
Done.